frontend/.env.example

```
@@ -1,6 +1,7 @@
 WORKOS_CLIENT_ID=""
 WORKOS_API_KEY=""
 NEXT_PUBLIC_WORKOS_REDIRECT_URI="http://localhost:3000/auth/callback"
+BASE_URL="http://localhost:3000"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)